src/components/ResultList/ListActions.tsx

```diff
@@ -94,7 +92,7 @@ export const ListActions = (props: IListActionsProps): ReactElement => {
   const handleOperationsLink = (operator: Operator) => {
     if (exploreAll) {
       // new search with operator
-      const q = `${operator}(${router.query.q as string})`;
+      const q = createOperatorQuery(operator, router.query.q as string);
```

Comment:
```suggestion
      const query = parseQueryFromUrl(router.query);
      const q = createOperatorQuery(operator, query.q);
```
Just so that the type is already set and you don't have to assert.